packages/recorder/src/controllers/fullcircle_api_controller.ts

```diff
@@ -13,7 +13,7 @@ export const initFullCircleApiRouter = ({sessionManager}: Deps) => {
     });
 
     fullcircleApiRouter.post('/record/stop', async (req, res) => {
-        const message = await sessionManager.finishCurrentSession();
+        const message = await sessionManager.finishCurrentSession('');
         res.send(message);
     });
```

Comment:
TODO: Explain in the readme how the sessions can be managed through this API endpoint. Hitting this endpoint is essentially the same thing as pressing enter in the terminal, except you also have to call `/record/start` to start a new session.

This was the first iteration of sessions, so it's currently outdated. Maybe we should have a single "save session" endpoint that accepts a session name so it can pass that to `finishCurrentSession`.